docs/getting-started-with-cosmosdb.md

```diff
@@ -0,0 +1,53 @@
+## Getting Started with Scalar DB on Cosmos DB
+
+## Overview
+This document briefly explains how you can get started with Scalar DB on Cosmos DB with a simple electronic money application.
+
+## Install prerequisites
+
+Scalar DB is written in Java. So the following software is required to run it.
+
+* [Oracle JDK 8](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) (OpenJDK 8) or higher
+* Other libraries used from the above are automatically installed through gradle
+
+## Cosmos DB setup
+Scalar DB uses Azure Cosmos DB for storage implementation. So Azure Cosmos DB account also required to run it.
+
+* [Azure Cosmos DB](https://docs.microsoft.com/en-us/azure/cosmos-db/introduction)
+    * Select **Azure Cosmos DB** service from the services on Azure web console.
+    * Select **Add**
+    * On the Create Azure Cosmos DB Account page, enter the basic settings for the new Azure Cosmos account.
```

Comment:
Can you highlight the page name?
Also, is it `Cosmos DB account` or `Cosmos account`? Please make it consistent.

Reply:
> Can you highlight the page name?

yes.

>  is it Cosmos DB account or Cosmos account? 

Cosmos DB account

The document updated accordingly.